abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+    - https://staging.continuum.io/prefect/fs/weasel-feedstock/pr1/a9226b3
```

Comment:
Could you build it for py312 too? If it's possible. I guess it will reduce our tech debt in the future. Because it's quite painful to rebuild the whole chain of spacy's downstream packages

Reply:
I can understand the pain. Let me know if I shall proceed, see previous comment: https://github.com/AnacondaRecipes/spacy-feedstock/pull/25#discussion_r1435351103

Reply:
See weasel PR w/ py312: https://github.com/AnacondaRecipes/weasel-feedstock/pull/1

Reply:
I've added required staging channels